tangelo/linq/simulator.py

```diff
@@ -230,6 +235,80 @@ def simulate(self, source_circuit, return_statevector=False, initial_statevector
                            if v > self.freq_threshold}
             return (frequencies, None)
 
+        elif self._target == "qiskit_device":
+            import qiskit
+            from qiskit.test import mock
+            from qiskit.providers.aer import AerSimulator
+            from qiskit.ignis.mitigation.measurement import complete_meas_cal, CompleteMeasFitter
```

Comment:
Qiskit Ignis has been deprecated in favour of qiskit_experiments etc (see readme at https://github.com/Qiskit/qiskit-ignis)